server/test/test_train_endpoint.py

```
@@ -74,21 +83,46 @@ def test_train_endpoint(tmp_path):
+        # ensure centroid model downloadable
         model_req = urllib.request.Request(
             f'http://localhost:{PORT}/latest-model',
             headers={'Authorization': 'Bearer testtoken'},
         )
         with urllib.request.urlopen(model_req) as mresp:
             assert mresp.getcode() == 200
-            model = json.loads(mresp.read().decode())
-            assert model.get("type") == "centroid_model"
+            data = json.loads(mresp.read().decode())
+            assert data.get('type') == 'centroid_model'
             assert "g1" in model.get("centroids", {})
```

Ruff (0.12.2)

Undefined name `model`

(F821)